Simplenote/NoteEditorViewController+Swift.swift

```diff
@@ -926,6 +926,18 @@ extension NoteEditorViewController {
             noteEditor.scrollToSelectedLocation()
         }
     }
+
+    @objc
+    func getRoundedFontSize() -> Int {
```

Comment:
Can we pleaaaase drop the`get`? (since it's not swift-y). Some of the old code, though, has that, buuut we should 🔥

Plus: perhaps `FontSettings` would be a more appropriate location?